.codecov.yml

```diff
@@ -0,0 +1,2 @@
+codecov:
```

Comment:
Do you need this config? Codecov should default to the default branch as given by github (at the moment `develop` but will be `master`)